workspaces/libnpmpublish/README.md

```diff
@@ -51,6 +51,14 @@ A couple of options of note:
   token for the registry. For other ways to pass in auth details, see the
   n-r-f docs.
 
+* `opts.provenance` - when running in a supported CI environment, will trigger
+  the generation of a signed provenance statement to be published alongside
+  the package. Mutually exclusive with the `provenanceFile` option.
+
+* `opts.provenanceFile` - specifies the path to an externally-generated
+  provenance statement to be published alongside the package. Mutually
+  exclusive with the `provenance` option.
```

Comment:
Should there be a note about the format of the provenance statement? Maybe also note that it should already be signed so that it's clear that npm isn't going to do signing for you?

Reply:
Yeah, good call. I'll update